app/handlers.go

```diff
@@ -0,0 +1,66 @@
+package app
+
+import (
+	"GOBootcamp/app/models"
+	"encoding/csv"
+	"fmt"
+	"log"
+	"net/http"
+	"os"
+	"strconv"
+)
+
+func (a *App) IndexHandler() http.HandlerFunc {
```

Comment:
You could remove the (a *App) as this is not being used, you should do this only when it is "part" of the structure, as this create a new handler it actually does not belong to App, in clean architecture, this would be part of it's own layer "package handler"